samples/ChangeFeedMigrationTool/DocumentFeedObserver.cs

```diff
@@ -0,0 +1,66 @@
+﻿//--------------------------------------------------------------------------------- 
+// <copyright file="DocumentFeedObserver.cs" company="Microsoft">
+// Microsoft (R)  Azure SDK 
+// Software Development Kit 
+//  
+// Copyright (c) Microsoft Corporation. All rights reserved.   
+// 
+// THIS CODE AND INFORMATION ARE PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,  
+// EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED WARRANTIES  
+// OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.  
+// </copyright>
+//---------------------------------------------------------------------------------
+
+namespace ChangeFeedMigrationSample
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Documents;
+    using Microsoft.Azure.Documents.ChangeFeedProcessor;
+    using Microsoft.Azure.Documents.Client;
+
+    /// <summary>
+    /// Cass to create instance of document feed observer. 
```

Comment:
The comment does not provide much information. Let's change it to something like this: This class is used to observe changes on change feed. ChangeFeedEventHost will create as many instances of this class as needed.